guides/cli.rst

```diff
@@ -241,7 +233,7 @@ The ``esphome <CONFIG> logs`` command validates the configuration and shows all
 
     Manually set the client id.
 
-.. option:: --serial-port SERIAL_PORT
+.. option:: --device SERIAL_PORT
 
-    Manually specify a serial port to use. For example ``/dev/cu.SLAB_USBtoUART``.
+    Manually specify a serial port/IP to use. For example ``/dev/cu.SLAB_USBtoUART``.
```

Comment:
Also worth mention OTA is an option

Reply:
I've added a note about OTA's, though not here (since this section is about logs ;))